samples/examples/image_classifier/train.md

```diff
@@ -0,0 +1,280 @@
+782/782 [=] - 385s 491ms/step - loss_ma: -2942.0786
+782/782 [=] - 334s 426ms/step - loss_ma: -3482.4651
+782/782 [=] - 331s 422ms/step - loss_ma: -3826.7684
+782/782 [=] - 328s 419ms/step - loss_ma: -4113.3548
+782/782 [=] - 324s 414ms/step - loss_ma: -4344.3997
+782/782 [=] - 356s 455ms/step - loss_ma: -4517.2223
+782/782 [=] - 340s 434ms/step - loss_ma: -4636.2948
+782/782 [=] - 338s 432ms/step - loss_ma: -4749.2001
+782/782 [=] - 332s 423ms/step - loss_ma: -4853.0608
+782/782 [=] - 333s 425ms/step - loss_ma: -4944.8810
+782/782 [=] - 326s 416ms/step - loss_ma: -5008.7755
+782/782 [=] - 326s 416ms/step - loss_ma: -5083.0938
+782/782 [=] - 315s 402ms/step - loss_ma: -5140.1222
+782/782 [=] - 311s 397ms/step - loss_ma: -5201.4831
+782/782 [=] - 316s 403ms/step - loss_ma: -5250.6943
+782/782 [=] - 318s 406ms/step - loss_ma: -5283.1387
+782/782 [=] - 316s 404ms/step - loss_ma: -5321.9071
+782/782 [=] - 320s 409ms/step - loss_ma: -5368.1527
+782/782 [=] - 321s 409ms/step - loss_ma: -5388.1669
+782/782 [=] - 317s 405ms/step - loss_ma: -5425.3241
+782/782 [=] - 319s 408ms/step - loss_ma: -5449.0800
+782/782 [=] - 320s 409ms/step - loss_ma: -5481.8549
+782/782 [=] - 319s 407ms/step - loss_ma: -5511.5501
+782/782 [=] - 318s 406ms/step - loss_ma: -5527.1451
+782/782 [=] - 319s 408ms/step - loss_ma: -5554.7566
+782/782 [=] - 373s 476ms/step - loss_ma: -5573.3425
+782/782 [=] - 376s 480ms/step - loss_ma: -5599.4756
+782/782 [=] - 375s 479ms/step - loss_ma: -5612.1058
+782/782 [=] - 375s 478ms/step - loss_ma: -5622.7508
+782/782 [=] - 373s 476ms/step - loss_ma: -5642.8896
+782/782 [=] - 374s 477ms/step - loss_ma: -5661.9375
+782/782 [=] - 373s 476ms/step - loss_ma: -5674.5788
+782/782 [=] - 374s 477ms/step - loss_ma: -5683.5199
+782/782 [=] - 373s 476ms/step - loss_ma: -5697.1596
+782/782 [=] - 375s 479ms/step - loss_ma: -5711.3716
+782/782 [=] - 376s 480ms/step - loss_ma: -5721.1513
+782/782 [=] - 377s 481ms/step - loss_ma: -5733.8188
+782/782 [=] - 376s 480ms/step - loss_ma: -5741.6455
+782/782 [=] - 376s 480ms/step - loss_ma: -5743.0460
+782/782 [=] - 376s 480ms/step - loss_ma: -5756.9715
+782/782 [=] - 376s 480ms/step - loss_ma: -5761.5570
+782/782 [=] - 376s 480ms/step - loss_ma: -5776.9497
+782/782 [=] - 376s 479ms/step - loss_ma: -5783.7933
+782/782 [=] - 374s 478ms/step - loss_ma: -5791.9981
+782/782 [=] - 377s 481ms/step - loss_ma: -5801.7900
+782/782 [=] - 377s 481ms/step - loss_ma: -5803.0183
+782/782 [=] - 376s 480ms/step - loss_ma: -5816.9740
+
+```
+
+Create a MLServer model settings file: `model-settings.json`:
+
+```json
+{
+  "name": "cifar10-outlier-detect",
+  "implementation": "mlserver_alibi_detect.AlibiDetectRuntime",
+  "parameters": {
+    "uri": "./",
+    "version": "v0.1.0"
+  }
+}
+
+```
+
+Save to local or remote storage the directory. Here we saved to Google Storage:
+
+```bash
+gsutil ls -R gs://seldon-models/mlserver/alibi-detect/cifar10-outlier
+
+gs://seldon-models/mlserver/alibi-detect/cifar10-outlier/:
```

Comment:
:question: This looks like it should be output but for some reason it's part of this shell block.  There's no comparison here as this file wasn't added previously, but I suspect it was probably already the case.

In any case, I'd hope it's reasonably obvious a user shouldn't copy/paste these lines, but rather that they're output.